repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -0,0 +1,53 @@
+package ammonite.repl
+
+import acyclic.file
+import java.io.{File, FileInputStream, IOException, FileWriter}
+import org.yaml.snakeyaml.Yaml
+import scala.collection.mutable.ArrayBuffer
+import scala.collection.JavaConversions.asScalaBuffer
+
+object Storage{
+  val dir = new java.io.File(System.getProperty("user.home") + "/.ammonite")    
+  if(dir.exists){
+    if(!dir.isDirectory){
+      dir.delete()
+      dir.mkdir()
+    }
+  } else {
+    dir.mkdir()
+  }
+
+  lazy val yaml = new Yaml
```

Comment:
Is initializing this expensive? If not let's create a new one in every `loadHistory` and `saveHistory` and save ourselves the messiness of caching + lazy-init + locking